app/controllers/journeys/adjustments/profitOrLoss/CheckNetProfitLossController.scala

```diff
@@ -42,21 +42,21 @@ class CheckNetProfitLossController @Inject() (override val messagesApi: Messages
     with Logging {
 
   def onPageLoad(taxYear: TaxYear, businessId: BusinessId): Action[AnyContent] = (identify andThen getData andThen requireData) { implicit request =>
-    val netAmount            = BigDecimal(-200)
-    val profitOrLoss         = returnProfitOrLoss(netAmount)
-    val formattedNetAmount   = formatSumMoneyNoNegative(List(netAmount))
-    val netProfitOrLossTable = buildNetProfitOrLossTable(profitOrLoss, 3000, 0.05, -3100)
-    val additionsTable       = buildAdditionsTable(profitOrLoss, 0, -0.05, 100.20)
-    val deductionsTable      = buildDeductionsTable(profitOrLoss, 200, -200.1)
+    val netAmount          = BigDecimal(-200)
+    val profitOrLoss       = returnProfitOrLoss(netAmount)
+    val formattedNetAmount = formatSumMoneyNoNegative(List(netAmount))
+    val table1             = buildTable1(profitOrLoss, 3000, 0.05, -3100)
+    val table2             = buildTable2(profitOrLoss, 0, -0.05, 100.20)
```

Comment:
Changed because table 2 and 3 swap the 'additions' or 'deductions' caption depending on if it's profit or loss